Client/Ecosia/UI/NTP/Cells/NTPImpactCell.swift

```diff
@@ -69,6 +69,14 @@ final class NTPImpactCell: UICollectionViewCell, NotificationThemeable, Reusable
         addProgress()
         addConstraints()
         applyTheme()
+        updateAccessibilityElements()
+    }
+    
+    func updateAccessibilityElements() {
+        container.isAccessibilityElement = true
+        container.shouldGroupAccessibilityChildren = true
+        container.accessibilityHint = "Open the Your Impact section"
```

Comment:
I wonder if we shouldn't anyway add to the L10N localisation files or at list the Strings file so it is not hardcoded and therefore harder to maintain. Maybe what could be done is also using existing texts somehow so they already have translations, but this will be trickier to do then...

Reply:
🤦 pushed without it for some reason! Will provide context as usual